packages/data/README.md

```diff
@@ -853,6 +853,10 @@ _Returns_
 
 -   `Object`: Data object returned by the `mapSelect` function.
 
+### waitForTransition
+
+Undocumented declaration.
```

Comment:
I ran `npm run docs:build` and got this, not sure why doc block is not parsed correctly yet.